dependencies.yaml

```diff
@@ -388,3 +396,12 @@ dependencies:
           - matrix: # All CUDA 11 versions
             packages:
               - cuda-python>=11.7.1,<12.0a0,<=11.8.3
+  test_java:
+    common:
+      - output_types: conda
+        packages:
+          - cxx-compiler
```

Comment:
cxx-compiler and cmake can be removed. We are already including the build-universal and build-cpp lists in the test_java file key, so they will be included that way.